src/Modules/Content/Dnn.PersonaBar.Pages/Pages.Web/src/components/PageHierarchy/pages.pageHierarchy.js

```diff
@@ -725,6 +725,7 @@ window.dnn.pages = window.dnn.pages || {};
                         sourceFind   = handler._findDataPosition(sourcePageId);
                         source       = viewModel.pagesList()[sourceFind.level].pages()[sourceFind.index];
                         sourceData   = handler._clonePageData(source);
+                        isDragItem = item.find('> div').hasClass('drag-item');
```

Comment:
is this only true if is new page? We may want put some different name so is clear the scenario where this got true and we perform the initialize request.